src/segments/sitecore.go

```diff
@@ -2,77 +2,96 @@ package segments
 
 import (
 	"encoding/json"
-	"path/filepath"
-	"strings"
+	"path"
 
 	"github.com/jandedobbeleer/oh-my-posh/src/platform"
 	"github.com/jandedobbeleer/oh-my-posh/src/properties"
 )
 
+const (
+	sitecoreFileName = "sitecore.json"
+	sitecoreFolderName = ".sitecore"
+	userFileName = "user.json"
+	defaultEnpointName = "default"
+)
+
 type Sitecore struct {
-	props properties.Properties
-	env   platform.Environment
+    props   	properties.Properties
+    env     	platform.Environment
+
+    EndpointName 	string
+	CmHost 			string
+}
 
-	Environment string
-	Cloud       bool
+type EndpointConfig struct {
+	Host string `json:"host"`
 }
 
-type SitecoreConfig struct {
-	Endpoints struct {
-		Default struct {
-			Ref        string `json:"ref"`
-			AllowWrite bool   `json:"allowWrite"`
-			Host       string `json:"host"`
-			Variables  struct {
-			} `json:"variables"`
-		} `json:"default"`
-	} `json:"endpoints"`
+type UserConfig struct {
+	DefaultEndpoint string 						`json:"defaultEndpoint"`
+	Endpoints 		map[string]EndpointConfig 	`json:"endpoints"`
 }
 
 func (s *Sitecore) Enabled() bool {
-	return s.shouldDisplay()
+	if s.env.HasFiles(sitecoreFileName) && s.env.HasFiles(path.Join(sitecoreFolderName, userFileName)) {
```

Comment:
this can be inverted so we do not indent the full block. Return false early and continue on the same level below.

Reply:
Thanks!
Fixed